docs/language-influences.rst

```diff
@@ -0,0 +1,25 @@
+###################
+Language Influences
+###################
+
+Solidity is a `curly-bracket language <https://en.wikipedia.org/wiki/List_of_programming_languages_by_type#Curly-bracket_languages>`_
+that has been influenced and inspired by several well-known programming languages.
+
+Solidity is most profoundly influenced by C++, but also borrowed concepts from languages like
+Python, JavaScript, and others.
+
```

Comment:
Maybe we should go a bit into detail about the influence from C++? Although I don't really know what to say there.

Reply:
``The influence from C++ can be seen in the syntax for variable declarations, for loops, the concept of overloading functions, implicit and explicit type conversions and many other details.``